test/unit/initialize.spec.ts

```diff
@@ -822,4 +823,56 @@ describe("Initialization", () => {
       expect(flicking.getPanelCount()).to.be.equals(2);
     });
   });
+
+  describe("bound", () => {
+    it("should clamp dest position when moving prev to adjacent panel #219", async () => {
+      // Given
+      flickingInfo = createFlicking(horizontal.panel30, {
+        bound: true,
+        anchor: 50,
+        hanger: 50,
+        threshold: 40,
+        defaultIndex: 6,
+        moveType: { type: "snap", count: 1},
+      });
+      const moves = [];
+      flickingInfo.instance.on("move", e => moves.push(e.axesEvent.pos.flick));
+
+      // When
+      simulate(flickingInfo.element, {
+        deltaX: 40,
+        duration: 200,
+      });
+      await waitFor(1000);
+
+      // Then
+      const scrollArea = (flickingInfo.instance as any).viewport.getScrollArea();
+      expect(moves.every(move => move <= scrollArea.next)).to.be.true;
```

Comment:
@WoodNeck Is it more meaningful to check `scrollArea.prev`?

Reply:
Because it's testing going out of bound when camera's at last panel to last - 1 panel.